<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serial_item</class>
 <widget class="QWidget" name="serial_item">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>82</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>397</width>
    <height>82</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>serial_item</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>sub</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_sub"/>
    </item>
    <item>
     <widget class="QPushButton" name="pasteButton_sub">
      <property name="text">
       <string>paste</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>video</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_video"/>
    </item>
    <item>
     <widget class="QPushButton" name="pasteButton_video">
      <property name="text">
       <string>paste</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_serial_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dexter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>season</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_season">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_episode">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>episode</string>
   </property>
  </widget>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>checkBox</zorder>
  <zorder>label_serial_title</zorder>
  <zorder>label_7</zorder>
  <zorder>spinBox_season</zorder>
  <zorder>spinBox_episode</zorder>
  <zorder>line</zorder>
  <zorder>label_8</zorder>
  <zorder>label_5</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
